matrix/rules.py

```diff
@@ -303,8 +305,10 @@ def exception_handler(self, context, loop=None, exc_ctx=None):
         reporter = functools.partial(self.exception_handler, context)
         self.loop.set_exception_handler(reporter)
         try:
+            await context.juju_model.connect_current()
```

Comment:
This is fine for now, but I think we'll want to expose controls in the future that let us do a model per test or model per suite.